pkg/utils/nodeclaim/nodeclaim_test.go

```diff
@@ -205,7 +205,7 @@ func TestGenerateNodeClaimManifest(t *testing.T) {
 		assert.Equal(t, nodeClaim.Labels[kaitov1alpha1.LabelWorkspaceNamespace], mockWorkspace.Namespace, "label must have same workspace namespace as workspace")
 		assert.Equal(t, nodeClaim.Labels[consts.LabelNodePool], consts.KaitoNodePoolName, "label must have same labels as workspace label selector")
 		assert.Equal(t, nodeClaim.Annotations[v1beta1.DoNotDisruptAnnotationKey], "true", "label must have do not disrupt annotation")
-		assert.Equal(t, len(nodeClaim.Spec.Requirements), 3, " NodeClaim must have 3 NodeSelector Requirements")
+		assert.Equal(t, len(nodeClaim.Spec.Requirements), 4, " NodeClaim must have 3 NodeSelector Requirements")
```

Comment:
I think if you change the previous to 4, I think the sentence should be " NodeClaim must have 3 NodeSelector Requirements".

Which one is the requirement you added?

Reply:
Fixed. And the requirement added was karpenter.k8s.aws/instance-gpu-count should be greater than 0 for aws